backend/fms_core/models/imported_file.py

```diff
@@ -13,6 +14,8 @@ class ImportedFile(TrackedModel):
     filename = models.CharField(max_length=500)
     # 100 characters allocated to the path length itself
     location = models.CharField(max_length=600)
+    added = models.DateTimeField(auto_now_add=True)
```

Comment:
i had removed those fields since they are totally redundant with the new ones. I had not removed the rest of the linked code though but we should think of removing them nonetheless unless this breaks something we can't handle.

Reply:
it has to be there so the migration can work (to transfer the fields from aded to created_at). otherwise it doesnt find the field as references

Reply:
This should not be the case ... We've removed many fields in that migration including fields we access (volume_history) to extract values. We should be able to delete those field from the model and still access the fields.